Anytype/Sources/Design system/InlinePicker/AnytypeInlinePicker.swift

```diff
@@ -33,9 +33,7 @@ public struct AnytypeInlinePicker<T> : View where T: (Identifiable & Equatable &
     func buildItem(value: T) -> some View {
         HStack {
             Spacer.fixedWidth(16)
-            Image(asset: .toastTick)
-                .renderingMode(.template)
-                .foregroundColor(Color.Dark.blue)
+            Image(asset: .system(name: "checkmark"))
```

Comment:
Why are you using icon not from design system?

Reply:
yes, current sharing extension implementation is based on iOS system components